grr/server/grr_response_server/bin/grrafana_test.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+# Lint as: python3
+"""Unittest for GRRafana HTTP server."""
+from absl.testing import absltest
+
+from grr_response_server.bin import grrafana
+
+# TODO(tsehori): We should implement tests for grrafana.py.
+class GRRafanaTest(absltest.TestCase):
```

Comment:
Nit: The test case should correspond to the class or function name it is testing. Since the tested class is named `Grrafana`, the test case name should be `GrrafanaTest`.